### ISSUE TITLE
Transcluded files opened as buffers.

### ISSUE THREAD

**Issue by dynameow** [nobiot/org-transclusion#170]
Hello, 
I'm not sure if this is a bug or the intended behavior that I've missed somewhere when reading the docs. I apologize if it's the latter.

I'm having an issue with transcluded files getting opened after toggling the org-transclusion minor mode. 
Apart from this problem, transclude works as intended. I've also tried the solutions mentioned in #19 just in case, as it seemed similar to my issue. 

![2023-03-04-022352_3000x1920_scrot](https://user-images.githubusercontent.com/25871701/222868259-9fcc3d08-13b9-4f73-b19e-34257f4ac7ef.png)
![2023-03-04-022405_3000x1920_scrot](https://user-images.githubusercontent.com/25871701/222868267-56fb1326-2e9b-411e-9c60-82a91f50ac84.png)

I'm using transclude on Doom Emacs with the default configuration. I've confirmed the issue is not caused by org-roam or my org configuration. 
Version info:
```$ .emacs.d/bin/doom version
GNU Emacs     v28.2            nil
Doom core     v3.0.0-pre       grafted, HEAD -> master, origin/master, origin/HEAD d5ccac5 2022-12-02 00:36:06 +0100
Doom modules  v22.10.0-pre     grafted, HEAD -> master, origin/master, origin/HEAD d5ccac5 2022-12-02 00:36:06 +0100```

**Comment by nobiot** [comment]
Can you check if it works without Doom, please?

There is also [known limitation](https://nobiot.github.io/org-transclusion/#Known-Limitations ) with Doom: 

Doom’s customization may interfere with Org-transclusion Refer to issue [#52](https://github.com/nobiot/org-transclusion/issues/52). The symptom is that in Doom you get an error message that includes this: “progn: ‘recenter’ing a window that does not display current-buffer.” Adding this in your configuration has been reported to fix the issue:
(advice-remove 'org-link-search '+org--recenter-after-follow-link-a)

It is probably rather drastic a measure. I will appreciate it if you find a less drastic way that works. Thank you.

**Comment by s4n1t** [comment]
Thanks for the response.
I've confirmed it does indeed work in vanilla emacs. I've also tried both vanilla doom emacs and spacemacs, and the issue occurs in both of them. 
Sadly, `(advice-remove 'org-link-search '+org--recenter-after-follow-link-a)` hasn't worked.

I don't have much experience with the emacs config nor lisp, but while trying different solutions I've discovered, that if I open org-agenda, org-transclusion suddenly works as intended, though I have no idea why this is the case.
